### DISCUSSION TITLE
Way to install on a different drive

### DISCUSSION THREAD

**Discussion by Kazevic** [winapps-org/winapps#226] [Support]
I use a combination of SSD + HDD, so ideally the Windows VM should be located on the HDD, but I can't figure out how to change it from var/lib/docker to something else. Is the only way to create a VM prior to that and adapt it to WinApps?

**Comment by oskardotglobal** [comment]
This is possible by using a bind mount. Note that you have to copy the old contents of the old named volume to the new bind mount unless you want to reinstall Windows.

**Reply by Kazevic** [reply]
Where is that? I don't see anything on compose.yaml, and I removed the VM before it finished the installation because it would crash my PC.
Thanks for the quick answer!
